atomix/cluster/src/test/java/io/atomix/raft/storage/log/RaftLogCommittedReaderTest.java

```diff
@@ -0,0 +1,152 @@
+/*
+ * Copyright © 2020 camunda services GmbH (info@camunda.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.atomix.raft.storage.log;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import io.atomix.raft.storage.log.RaftLogReader.Mode;
+import io.atomix.raft.storage.log.entry.ApplicationEntry;
+import io.atomix.raft.storage.log.entry.RaftLogEntry;
+import java.io.File;
+import java.nio.ByteBuffer;
+import java.nio.ByteOrder;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.io.TempDir;
+
+class RaftLogCommittedReaderTest {
+
+  @TempDir File directory;
+
+  private RaftLog raftlog;
+  private RaftLogReader committedReader;
+
+  private final ByteBuffer data = ByteBuffer.allocate(8);
+
+  @BeforeEach
+  void setup() {
+    raftlog = RaftLog.builder().withDirectory(directory).withName("test").build();
+    committedReader = raftlog.openReader(Mode.COMMITS);
+    data.order(ByteOrder.LITTLE_ENDIAN).putInt(123456);
```

Comment:
Nit: the `order` can be moved at declaration time, and if we only set a static value, can also be done at initialization.